src/pipe_manager.cpp

```diff
@@ -1044,6 +1044,10 @@ void on_registry_global(void* data,
         nd->nd_info->media_class = media_class;
         nd->nd_info->name = name;
 
+        if (const auto* serial = spa_dict_lookup(props, PW_KEY_OBJECT_SERIAL)) {
```

Comment:
Not sure if it's really a bug, but when compiling the flatpak with PW 0.3.31 I got:
```
../src/pipe_manager.cpp: In function ‘void {anonymous}::on_registry_global(void*, uint32_t, uint32_t, const char*, uint32_t, const spa_dict*)’:
../src/pipe_manager.cpp:1047:57: error: ‘PW_KEY_OBJECT_SERIAL’ was not declared in this scope; did you mean ‘PW_KEY_DEVICE_SERIAL’?
```

Bumping the bundled PW lib to 0.3.47 fixed it. 0.3.31 was bundled in the past for better compatibility, but things have [improved](https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/1515) upstream. Perhaps I will reinvestigate a better solution for https://github.com/wwmm/easyeffects/issues/1089...